update fermi mission page for flare (#3692)
nasa-gcn/gcn.nasa.gov: app/routes/missions.fermi/route.mdx

</WithFeature>

<WithFeature LAT_JSON>

**JSON-Serialized Fermi/LAT Flare Notices in GCN Kafka:**

Fermi/LAT distributes JSON-serialized notices for gamma-ray flares detected by the LAT Flare Advocate pipeline on the GCN Kafka topic `gcn.notices.fermi.lat`.

> 

Author: ```suggestion

Currently, flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is at least 10 times the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) greater than 30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is less than 25 arcmin.
```
@ChiaraBartolini96 based on discussion I have added the comment. Please confirm it.

Reviewer: Perfect! I confirm everything you wrote. Thank you very much @Vidushi-GitHub.

</WithFeature>

<WithFeature LAT_JSON>

**JSON-Serialized Fermi/LAT Flare Notices in GCN Kafka:**

Fermi/LAT distributes JSON-serialized notices for gamma-ray flares detected by the LAT Flare Advocate pipeline on the GCN Kafka topic `gcn.notices.fermi.lat`.

Currently, flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is at least 10 times the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) greater than 30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is less than 25 arcmin.

> For schemas and examples see [/schema/stable/gcn/notices/fermi/lat](/schema/stable/gcn/notices/fermi/lat) or use the [Schema Browser](/docs/schema/stable/gcn/notices/fermi/lat/).

Author: ```suggestion

For schemas and examples see [/schema/stable/gcn/notices/fermi/lat](/schema/stable/gcn/notices/fermi/lat) or use the [Schema Browser](/docs/schema/stable/gcn/notices/fermi/lat/).
```

</WithFeature>

<WithFeature LAT_JSON>

**JSON-Serialized Fermi/LAT Flare Notices in GCN Kafka:**

> Fermi/LAT distributes JSON-serialized notices for gamma-ray flares detected by the LAT Flare Advocate pipeline on the GCN Kafka topic `gcn.notices.fermi.lat`.

Reviewer 2: ```suggestion
Fermi/LAT distributes JSON-serialized notices for gamma-ray flares detected by the LAT Flare Advocate pipeline on the GCN Kafka topic `gcn.notices.fermi.lat.flare`.
```

Author: Directory name is `gcn.notices.fermi.lat`, flare is schema name. Need to cross-check the topic name.

Author: @dakota002 please confirm the topic name

> Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is >10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) >30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is <25 arcmin.

Reviewer 2: ```suggestion
Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is >10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) >30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is <25 arcmin.  These are generated once per day with analysis on the prior 24 hour interval.
```

> Currently, flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is at least 10 times the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) greater than 30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is less than 25 arcmin.

Reviewer 2: ```suggestion
Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is >10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) >30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is <25 arcmin.
```

> Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is >10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) >30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is <25 arcmin.  These are generated once per day with analysis on the prior 24 hour interval.

Reviewer 2: ```suggestion
Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is >10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio ≥ 10), the detection has a Test Statistic (TS) >30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is <25 arcmin.  These are generated once per day with analysis on the prior 24 hour and 6 hour intervals.
```

> Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is \>10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio $\geq$ 10), the detection has a Test Statistic (TS) \>30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is \<25 arcmin. These are generated once per day with analysis on the prior 24 hour and 6 hour intervals.

Author: ```suggestion
Flare notices are issued only for known extragalactic sources. A source is considered to be in a flare state if its observed flux is at least 10x the flux reported in the 4FGL-DR4 catalog (i.e., a flux ratio $\geq$ 10), the detection has a Test Statistic (TS) \>30, and the angular separation between the detected photons and the nearest 4FGL-DR4 source is \<25 arcmin. These are generated once per day with analysis on the prior 24 hour and 6 hour intervals.
```

> For schemas and examples see [/schema/stable/gcn/notices/fermi/lat](/schema/stable/gcn/notices/fermi/lat) or use the [Schema Browser](/docs/schema/stable/gcn/notices/fermi/lat/).

Reviewer 2: These aren't yet in the Schema Browser. Do we need to do something to add them? @dakota002
